docs/DeveloperGuide.md

```diff
@@ -211,6 +211,38 @@ The steps above are summarised using a sequence diagram as shown below.
     * Pros: Easy to extend and manipulate attributes of a player.
     * Cons: Hard to implement.
 
+### 4.3 Add tags feature
+
+#### 4.3.1 Implementation
+This feature allows the user to add tags to contacts in the list. It is facilitated by `ModelManager` which
+makes use of the method `#setPerson()` and `#updateFilteredPersonList()` to add tags to a contact.
+
+Given below is an example usage scenario of how the add tag mechanism behaves at each step.
+
+Step 1: The user inputs `tag-add 1 t/friend` to add the tag "friend" to the first contact in the list.
```

Comment:
Should the command format be tag-add-p 1 friend instead?

Reply:
Fixed